horace_core/sqw/coord_transform/@spher_proj/private/check_combo_arg_.m

```diff
@@ -0,0 +1,30 @@
+function obj = check_combo_arg_(obj)
+% Check validity of interdependent fields
+%
+%   >> obj = check_combo_arg(w)
+%
+% Throws HORACE:spher_proj:invalid_argument with the message
+% suggesting the reason for failure if the inputs are incorret
+% w.r.t. each other.
+%
+% Normalizes input vectors to unity and constructs the
+% transformation to new coordinate system when operation is
+% successful
+%
+
+ez = obj.ez/norm(obj.ez);
```

Comment:
have we checked that obj.ez has nonzero norm?

Reply:
yes, it has been checked on 0 before at setting it, using `check_3vector`